build-tools/packages/version-tools/src/semver.ts

```diff
@@ -50,8 +50,8 @@ export function bumpRange(
 				bumpType === "current"
 					? originalNoPrerelease
 					: scheme === "virtualPatch"
-					? bumpVersionScheme(originalNoPrerelease, bumpType, "virtualPatch")
-					: semver.inc(originalNoPrerelease, bumpType);
+					  ? bumpVersionScheme(originalNoPrerelease, bumpType, "virtualPatch")
```

Comment:
🥳🥳🥳🥳🥳🥳🥳🥳

Reply:
I mean, prettier has fixed this too in a more recent version. But I'm just happy to see sane ternary indentation 😭